packages/app/package.json

```diff
@@ -17,6 +17,7 @@
     "test:coverage": "jest --collect-coverage",
     "test:watch": "jest --watch",
     "typecheck": "tsc --noEmit",
+    "version": "yarn",
```

Comment:
This is a workaround to [this bug](https://github.com/lerna/lerna/issues/1171) in Lerna.